GiniVision/Classes/OnboardingPage.swift

```diff
@@ -84,9 +84,9 @@ import UIKit
         // Image view
         imageView.translatesAutoresizingMaskIntoConstraints = false
         ConstraintUtils.addActiveConstraint(item: imageView, attribute: .top, relatedBy: .equal, toItem: contentView, attribute: .top, multiplier: 1, constant: 0)
-        ConstraintUtils.addActiveConstraint(item: imageView, attribute: .width, relatedBy: .lessThanOrEqual, toItem: nil, attribute: .width, multiplier: 1, constant: 204)
+        ConstraintUtils.addActiveConstraint(item: imageView, attribute: .width, relatedBy: .lessThanOrEqual, toItem: nil, attribute: .width, multiplier: 1, constant: 612)
```

Comment:
It is three times the old value (the old value is taken from the max image width size). I have tested with twice the old value on iPad and I think it looks better being three times the old value.
Following the same criteria from the iPhone, it should be 810 instead of 612 but I tested it and IMO It looks too big.

Anyway, the problem it is not with the max value (612) but with the mini which has not been modified.